src/Iddict.elm

```diff
@@ -118,13 +118,16 @@ empty =
 -}
 fromList : List ( Int, a ) -> Iddict a
 fromList items =
+    let
+        ( cursor, dict ) =
+            List.foldl
+                (\( k, v ) ( cur, acc ) -> ( max (k + 1) cur, Dict.insert k v acc ))
+                ( 0, Dict.empty )
+                items
+    in
     Iddict
-        { cursor =
-            items
-                |> List.map Tuple.first
-                |> List.maximum
-                |> Maybe.withDefault 0
```

Comment:
It seems like a mistake of forgetting to add `|> (+) 1` on a new line to up the cursor one above the maximum.

What makes you prefer a different implementation?